solc_json_parser/parser.py

```diff
@@ -411,7 +435,7 @@ def _add_inherited_function_fields(data_dict: Dict[int, ContractData]):
                             new_function.inherited_from = base_contract_name
                             contract.functions.append(new_function)
 
-        # self.save_solc_ast_json("Storage_solc_ast_no_abstractBugC")
+        # self.save_solc_ast_json("event")
```

Comment:
please remove unused code if no longer useful

Reply:
This is useful when developing new features or debugging.